src/test/scala/ai/privado/languageEngine/ruby/download/ExternalDependenciesDownloadTestBase.scala

```diff
@@ -0,0 +1,44 @@
+package ai.privado.languageEngine.ruby.download
+
+import ai.privado.cache.RuleCache
+import better.files.File
+import io.joern.rubysrc2cpg.{Config, RubySrc2Cpg}
+import io.joern.x2cpg.X2Cpg.applyDefaultOverlays
+import io.shiftleft.codepropertygraph.generated.Cpg
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+abstract class ExternalDependenciesDownloadTestBase extends AnyWordSpec with Matchers with BeforeAndAfterAll {
```

Comment:
Instead of having specific test base class, we can probably use the rubyCode2CpgFixture in Joern

Reply:
not able to find this `rubyCode2CpgFixture`.